core/src/main/java/org/apache/carbondata/core/readcommitter/CommitedIndexFilesUpdatedTimestamp.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.core.readcommitter;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CommitedIndexFilesUpdatedTimestamp implements Serializable {
+
+  private Map<Long, Integer> updatedTimestampMap = new HashMap<>();
```

Comment:
It always has 2 attributes, not a map,
1. timestamp : It has max modified time across all files in a segment.
2. count.: the total number of files in a segment.